src/components/sky-nav/sky-nav.ts

```diff
@@ -13,55 +13,107 @@ import tokens from '../../compiled/tokens/js/tokens';
  */
 export const initSkyNav = (navButton: HTMLButtonElement) => {
   const menu = navButton.nextElementSibling as HTMLElement;
+  const navWrapper = navButton.closest('.c-sky-nav') as HTMLElement; // Review: should this use a .js- class or some different selector?
```

Comment:
> Review: should this use a .js- class or some different selector?

Good call out. Unless we've changed our conventions, this should use a `.js-` CSS class.

Reply:
Or...hmm...now I'm not sure. 🤔 

What do you think, @Paul-Hebert or @spaceninja?

Reply:
Am I correct in understanding before we didn't need to worry about the wrapper element which is why we never had to make this decision in the past? 🔍 👀

Reply:
Hmm, good question. If we need to target it with JS I'd lean towards adding a `js-` class.

Reply:
That makes sense, thanks, @Paul-Hebert. I am now leaning the same way. 

We should also update the [documentation](https://v-next--cloudfour-patterns.netlify.app/?path=/docs/components-sky-nav--dark#javascript-instructions) as part of this change.

It's still a bit odd because we initialize on the button when calling `initSkyNav()` but the wrapper is also important and targeted via JS. If we document it well, it should be fine, though.